modules/mo_pytorch/mo_extensions/front/pytorch/detection_output_ext.py

```diff
@@ -28,13 +28,22 @@ class DetectionOutputExtractor(FrontExtractorOp):
 
     @classmethod
     def extract(cls, node):
-        attrs = {
-            'variance_encoded_in_target': int(node.module.variance_encoded_in_target),
-            'nms_threshold': node.module.nms_threshold,
-            'confidence_threshold': node.module.confidence_threshold,
-            'top_k': node.module.top_k,
-            'keep_top_k': node.module.keep_top_k,
-            'code_type': node.module.code_type,
-        }
+        attr_names = [
+            'variance_encoded_in_target',
+            'nms_threshold',
+            'confidence_threshold',
+            'top_k',
+            'keep_top_k',
+            'code_type',
+            'share_location',
+            'background_label_id',
+            'clip_before_nms',
+        ]
+
+        attrs = {}
+        for attr in attr_names:
+            if hasattr(node.module, attr):
+                attrs[attr] = getattr(node.module, attr)
```

Comment:
We had a cast to `int` for the `variance_encoded_in_target` attribute before. Don't we need this cast anymore?

Reply:
Yes, seems like we don't need it anymore. At least tested with OpenVINO 2021.4 and master branch.